### PR TITLE
Feature/blueduc/plantingapiv1

### PR DESCRIPTION
**Pull request by blueduc**
code that can pass travis CI

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: python
+python:
+  - "3.6"
+script:
+  - pep8 --exclude=migrations .
```
Comment (rfkimi) [inline .travis.yml]: use pycodestyle instead of pep8

File: planting/planting_api_v1.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env python
-# encoding: utf-8
+# -*- coding: utf -*-
```
Comment (rfkimi) [inline planting/planting_api_v1.py:2]: it's a mistake

File: planting/planting_module/network/__init__.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-# encoding: utf-8
+# -*- coding: utf8 -*-# -*- coding: utf8 -*-
```
Comment (rfkimi) [inline planting/planting_module/network/__init__.py]: ???

File: planting/planting_api_v1.py (hunk 2 above)
Comment (blueduc) [reply planting/planting_api_v1.py:2]: its edit by mark, who said its a better style of coding

Comment (rfkimi) [reply planting/planting_api_v1.py:2]: ok

File: .travis.yml (hunk 1 above)
Comment (blueduc) [reply .travis.yml]: https://pypi.org/project/pycodestyle/
maybe they are the same thing

File: planting/planting_module/network/__init__.py (hunk 3 above)
Comment (blueduc) [reply planting/planting_module/network/__init__.py]: > ???
...

Comment (rfkimi) [reply planting/planting_module/network/__init__.py]: why write it twice？

File: tests/test_operation.py [hunk 4]
```diff
@@ -20,7 +20,8 @@ def test_copy(machine: Machine):
 
 def test_download(machine: Machine):
     assert True is machine.download(
-        url="http://www.runoob.com/wp-content/uploads/2015/10/vi-vim-cheat-sheet-sch.gif",
+        url="http://www.runoob.com/wp-content/uploads/ \
+        2015/10/vi-vim-cheat-sheet-sch.gif",
```
Comment (rfkimi) [inline tests/test_operation.py]: url="http://www.runoob.com/wp-content/uploads/" \
        "2015/10/vi-vim-cheat-sheet-sch.gif",

**Pull request merged**